core/bones/date.py

```diff
@@ -33,24 +32,25 @@ def __init__(
         """
         super().__init__(**kwargs)
 
-        if creationMagic or updateMagic:
-            self.readonly = True
-
-        self.creationMagic = creationMagic
-        self.updateMagic = updateMagic
-
         if not (date or time):
             raise ValueError("Attempt to create an empty datebone! Set date or time to True!")
 
-        if localize and not (date and time):
-            raise ValueError("Localization is only possible with date and time!")
+        if creationMagic or updateMagic:
+            self.readonly = True
```

Comment:
I really don't know why this is defined... anyway, it's "backward" compatible, still.